routes/productDetail.js

```diff
@@ -0,0 +1,59 @@
+const express = require("express");
+const router = express.Router();
+const Product = require("../models/Product");
+const User = require("../models/User");
+
+// POST '/api/products' => Create product
+router.post("/", (req, res) => {
+  console.log("[productDetail.js]: POST route");
+  const {
+    title,
+    description,
+    imageUrl,
+    brand,
+    category,
+    quantity,
+    price,
+    currency,
+    tags,
+    company,
+    latitude,
```

Comment:
this needs to be modified according to current model schema where lat long is nested under location. I can look at this once google maps is setup